scripts/package.js

```diff
@@ -1,27 +1,10 @@
 const exec = require('child_process').exec
 const fs = require('fs')
-const fse = require('fs-extra')
 
-fs.rmSync('package', { recursive: true, force: true })
+fs.rmSync('lib', { recursive: true, force: true })
 
-exec('mkdir package && cd package && mkdir lib && babel --config-file ../package.babel.js ../src -d lib --extensions .js,.jsx,.snap --copy-files --no-copy-ignored', (err) => {
+exec('mkdir lib && babel --config-file ./package.babel.js ./src -d lib --extensions .js,.jsx,.snap --copy-files --no-copy-ignored', (err) => {
```

Comment:
is it common naming `lib`? maybe we will leave only `package`? vice verca?

Reply:
as I noticed, usually folder in npm regestry is called 'lib', so I decided to set the same name